src/Microsoft.DotNet.Build.Tasks/Targets/post-process-assembly.targets

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <IntermediateAssemblyPath>$(IntermediateOutputPath)$(TargetName)$(TargetExt)</IntermediateAssemblyPath>
+    <PostProcessAssemblyMarker>$(IntermediateAssemblyPath).postprocessed</PostProcessAssemblyMarker>
+  </PropertyGroup>
+
+  <Target Name="PostProcessAssembly"
+    AfterTargets="CoreCompile"
+    Inputs="$(IntermediateAssemblyPath)"
+    Outputs="$(PostProcessAssemblyMarker)">
+
+    <CallTarget Targets="@(PostProcessAssemblySteps)" />
```

Comment:
Yeah, I went with this approach because, basically, FillPartialFacade and OpenSourceSign were conflicting with each other, especially in incremental builds. They essentially have the same inputs and outputs, so each would trigger the other in turn. Additionally, OpenSourceSign fails if the assembly is already signed, so another target modifying the assembly but not "unsigning" it (as FillPartialFacade does) would cause the signing target to fail. I tried to generalize the solution here.